editor/src/components/canvas/canvas-component-entry.tsx

```diff
@@ -42,12 +42,12 @@ export const CanvasComponentEntry = betterReactMemo(
       return null
     } else {
       return (
-        <CanvasErrorBoundary uiFilePath={canvasProps.uiFilePath} reportError={props.reportError}>
-          <UiJsxCanvas
-            {...canvasProps}
-            clearErrors={props.clearErrors}
-            reportError={props.reportError}
-          />
+        <CanvasErrorBoundary
+          uiFilePath={canvasProps.uiFilePath}
+          reportError={props.reportError}
+          topLevelElementsIncludingScenes={canvasProps.topLevelElementsIncludingScenes}
```

Comment:
I wonder if anything else needs to go in there like `imports` and `jsxFactoryFunction`, might be worth picking through `UiJsxCanvasProps`.

Reply:
I was thinking we'd be best off using the raw code from the file here tbh

Reply:
Or that.

Reply:
+1 to raw code

Reply:
Done